Classes/Domain/Service/ZoomApiService.php

```diff
@@ -110,29 +115,30 @@ public function getUpcomingMeetings(bool $skipCache = false): array
      */
     public function getRecordings(DateTime|string $from, DateTime|string $to, bool $skipCache = false): array
     {
-        $cacheEntryIdentifier = sprintf('recordings_%s_%s', is_string($from) ? $from : $from->format('Y-m-d'), is_string($to) ? $to : $to->format('Y-m-d'));
-
-        $recordings = $this->getCacheEntry($cacheEntryIdentifier);
-        if (!$skipCache && $recordings !== false) {
-            return $recordings;
-        }
-
         if (is_string($from)) {
             $from = new DateTimeImmutable($from);
         } else {
             $from = DateTimeImmutable::createFromMutable($from);
         }
+        assert($from instanceof DateTimeImmutable);
```

Comment:
Is this really necessary her? 
At that point, `$from` must be a `DateTimeImmutable` (either from a `string`, or a `DateTime`), no?.